### ISSUE TITLE
Package binaries for all releases?

### ISSUE THREAD

**Issue by jfeltesse-mdsol** [gjcarneiro/yacron#37]
Hi,

Thanks for this tool!

Release 0.10.0 featured a binary, which was really helpful in installing yacron without having to deal with a base image's installed python version.

Could newer releases also be packaged that way and published?

For reference, this is how we pull yacron:

```
ARG YACRON_BASE_URL=https://github.com/gjcarneiro/yacron/releases/download
ARG YACRON_VERSION=0.10.0
RUN curl -sSL -o yacron.xz $YACRON_BASE_URL/$YACRON_VERSION/yacron-$YACRON_VERSION-x86_64-unknown-linux-gnu.xz && \
  unxz yacron.xz && \
  chmod +x yacron
```

**Comment by gjcarneiro** [comment]
Yes, I understand that, I really wish I could, but I found out that the binary was broken.  Sentry reporting was not working.  And who knows what else might be broken as well that I don't know about.  I spent a lot of time trying to fix it, in the end I opted to rather stop distributing the binary rather than ship a broken binary, see #34. 

I'll keep this open for now.  I'd like to ship a binary, but a bug-free binary is not easy to achieve.
